joss-paper/paper.md

```diff
@@ -89,15 +89,34 @@ In contrast, the C compiler is more adapted to support GPU tools such as CUDA, a
 
 Pyccel is designed for two different use cases:
 (1) accelerate Python code by converting it to Fortran and providing a CPython wrapper to interface between the low-level and high level languages,
-(2) generate low-level Fortran code from Python code.
+(2) generate low-level C or Fortran code from Python code.
 The latter case follows from the fact that the code is human-readable.
 This means that Pyccel can also be used to simplify the process of going from a prototype (which is often written in inefficient languages which are quick to write) to production code (written in a low-level language).
 To this end, Pyccel is designed to allow the use of low-level legacy codes and some Python scientific libraries such as numpy, scipy, etc.
 
 # Benchmarks
 
+A few example codes are used to provide an indication of the performance of Pyccel as compared to the popular accelerators Numba [@NUMBA] and Pythran [@Pythran2015].
+The source code can be found in \url{github.com/pyccel/pyccel-benchmarks}.
+These examples, which illustrate several common scientific computing problems, are based on open-source example codes [@JBurkhardt; @CFD].
```

Comment:
The word "example(s)" is repeated in the same sentence. Maybe use a synonym?

Reply:
f2dae81b